posts/2022-02-08-feature-generation-tech-preview.adoc

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "Introducing automatic generation of features in Liberty server configuration (Tech Preview)"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/kathrynkodama
+author_github: https://github.com/kathrynkodama
+seo-title: Introducing automatic generation of features in Liberty server configuration (Tech Preview) - OpenLiberty.io
+seo-description: Tech Preview - Open liberty's dev mode automatically generates Liberty server configuration features as you iteratively develop your cloud-native Java applications.
+blog_description: "Tech Preview - Open liberty's dev mode automatically generates Liberty server configuration features as you iteratively develop your cloud-native Java applications."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Introducing automatic generation of features in Liberty server configuration (Tech Preview)
+Kathryn Kodama <https://github.com/kathrynkodama>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+In this post, we introduce a technology preview for automatically generating a list of features in your project's Liberty server configuration (server.xml). As part of Liberty dev mode, features will now be automatically generated in your server configuration as you develop your cloud-native Java application.
```

Comment:
I'm not sure if we should have "server.xml" in brackets here. Technically, `generated-features.xml` is an extension of `server.xml`, but I wonder if this would imply to users that the list shows up in their `server.xml`. Thoughts?

Reply:
I think (server.xml) just clarifies that by Liberty server configuration we mean the server.xml, since users may be more familiar with that file name.